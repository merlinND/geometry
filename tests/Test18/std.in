C myCircle 0 0 1
R myRectangle 2 0 0 1
OA myOA2 myCircle
L myLine 0 0 1 1
OA myOA1 myCircle myRectangle myLine myOA2
EXIT
