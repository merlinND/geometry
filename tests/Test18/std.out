C myCircle 0 0 1
OK
R myRectangle 2 0 0 1
OK
OA myOA2
OK
L myLine 0 0 1 1
OK
OA myCircle myRectangle myLine myOA2
OK
